R/render.R

```diff
@@ -66,14 +69,35 @@ moon_reader = function(
 
   play_js = if (is.numeric(autoplay <- nature[['autoplay']]) && autoplay > 0)
     sprintf('setInterval(function() {slideshow.gotoNextSlide();}, %d);', autoplay)
+
+  countdown_js = NULL
+  if (!is.null(countdown <- nature[['countdown']]) && countdown) {
+
+    countdown_js_raw <- "var c = setInterval(function(){}, 1000); var tot = slideshow.getSlideCount(); slideshow.on('afterShowSlide', function (slide) {var time_left = %d; var counter_div = document.getElementsByClassName('remark-slide-number').item(slide.getSlideIndex()); var cur = counter_div.innerHTML; function counter() { return window.setInterval(function() { time_left = time_left - 1000; counter_div.innerHTML = time_left/1000 + 's ' + (slide.getSlideIndex()+1) + ' / ' + tot; if (time_left < 0) counter_div.style.color = 'red';}, 1000);} clearInterval(c); c = counter();})"
+
+    if (is.numeric(autoplay <- nature[['autoplay']]) && autoplay > 0) {
+      countdown_js = sprintf(countdown_js_raw, autoplay)
```

Comment:
You probably misunderstood the `autoplay` option. It specifies the display time for each page instead of the total display time of all pages. In other words, `countdown = autoplay * the number of pages`.

I think it makes sense and will be more clear not to associate these two options.